### PR TITLE
Address leftover log names after concurrent merges

### PR DESCRIPTION
**Pull request by lucafavatella**
Please refer to commit message for details.

### CODE REVIEW

**Comment by tolbrino** [comment]
This is good once it is rebased on #2173.

**Comment by lucafavatella** [comment]
It is uncler whether setting log level from user config works for the pow log file. [Tracked](https://www.pivotaltracker.com/story/show/164056695).

File: config/dev.config [hunk 1]
```diff
@@ -58,15 +58,6 @@
                ]}
              ]}
            ]},
-           {epoch_pow_cuckoo_lager_event, [
```
Comment (jur0) [inline config/dev.config]: @lucafavatella have you actually tried the logs are still there? My understanding was that this handler should stay and the traces are redirected into it based on the file `log/aeternity_pow_cuckoo.log` in the handler and the same must be set for `lager_file_backend` in the trace.

Comment (tolbrino) [reply config/dev.config]: Traces are essentially redirections of log output. However, the sink is defined as part of the trace definition and must not exist before.

Comment (lucafavatella) [reply config/dev.config]: > have you actually tried the logs are still there?

@jur0 I did not check manually. This was based on code inspection.

I am giving priority to other PRs.

Comment (hanssv) [reply config/dev.config]: I checkout the branch, made a prod-package, started it with mining and got the expected log file (and only the expected log file) including content.

**Pull request merged**